### ISSUE TITLE
Yomichan getting stuck on profile selection

### ISSUE THREAD

**Issue by byjtje** [FooSoft/yomichan#1219]
**Description**
Pressing X whilst in the profile selection of the pop-up (the one which appears in the bottom right corner when viewing a kanji dictionary item) causes the following pop-ups to still have the profile selection overlay, whilst no longer having any buttons to remove it. This makes Yomichan unuseable until the page is reloaded/switched.

**Browser version**
Firefox 84.0.1

**Yomichan version**
20.12.31.0

**Comment by toasted-nutbread** [comment]
Thanks for reporting; I've created a fix which I will included in the next release/patch release.

**Comment by byjtje** [comment]
> Thanks for reporting; I've created a fix which I will included in the next release/patch release.

I have to thank you!
On a sidenote, the action bar remains even after going back, not sure if it's intentional. Also currently the only way to completely hide the action bar is by editing the custom CSS, think it wouldn't be bad to also include that as an option.

**Comment by toasted-nutbread** [comment]
It is intentional, as you can navigate forward, same as browser history. Once there is history, the sidebar will remain visible until the history is cleared (new text scanned / popup closed).

More options could be added for whether the sidebar is visible, yes; would you be expecting the sidebar to not show up, even when back history is present?

**Comment by byjtje** [comment]
>would you be expecting the sidebar to not show up, even when back history is present?

Sorry I made a mistake. Due it being on the right side I thought it was a new feature which just took away space, but after removing it I noticed that this is the smaller bar we had at the top the previous versions – and that removing it would also remove the ability to go back from a lone kanji definition. Although I would say that it being on the top/bottom is a more sensible default, as with default options horizontal space is a bigger problem and having the bar there causes many additional lines (see picture)
![newline](https://user-images.githubusercontent.com/14181316/104136034-9331b500-538b-11eb-9ce8-bd453dc13acd.png)


In hindsight of that I think there's no need for a removal option, sorry for the bother.
